application/models/Airports.php

```diff
@@ -3,13 +3,55 @@
 /**
  * Dummy model class that represents the airports our airline services
  */
-class Airports extends CSV_Model
+class Airports extends CI_Model
 {
+    private $baseId;
+    private $codesWeService;
 
     // Constructor
     public function __construct()
     {
         parent::__construct(APPPATH . DATA_AIRPORTS, 'id');
```

Comment:
We don't need this anymore. Should get rid of the constant definition as well in [`application/config/constants.php`](https://github.com/bigspennyj/wacky-webapp/blob/23a8c3b8af082c3f4c166309fa72987eb62980e1/application/config/constants.php#L87)